packages/playwright-core/src/server/android/android.ts

```diff
@@ -264,9 +264,13 @@ export class AndroidDevice extends SdkObject {
     debug('pw:android')('Force-stopping', pkg);
     await this._backend.runCommand(`shell:am force-stop ${pkg}`);
     const socketName = isUnderTest() ? 'webview_devtools_remote_playwright_test' : ('playwright-' + createGuid());
-    const commandLine = this._defaultArgs(options, socketName).join(' ');
+    const commandLine = this._defaultArgs(options, socketName).map(arg => {
+      if (!arg.includes('\'') && !arg.includes('"'))
+        return `"${arg}"`;
```

Comment:
What are you trying to do here, quote the strings? If so, you should quote and escape.